pom.xml

```diff
@@ -146,13 +147,48 @@ along with this program.  If not, see <http://www.gnu.org/licenses />.
                  <artifactId>guava</artifactId>
                  <version>14.0.1</version>
             </dependency>
+            <dependency>
+                <groupId>org.ebaysf.web</groupId>
+                <artifactId>cors-filter</artifactId>
+                <version>1.0.0</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>javax.servlet</groupId>
+                        <artifactId>servlet-api</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+            <dependency>
+                <groupId>org.hamcrest</groupId>
+                <artifactId>hamcrest-core</artifactId>
+                <version>1.3</version>
+                <!-- Must leave compile scope to prevent Mockito's transitive, conflicting, runtime
```

Comment:
Could the conflicting version be excluded here and explicit scope be set instead of a comment nobody will read?